gp-advanced-save-and-continue/gpasc-use-submission-values-for-draft-names.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/**
+ * Gravity Perks // Advanced Save and Continue // Use Submission Values for Draft Names
+ *
+ * https://gravitywiz.com/documentation/gravity-forms-advanced-save-continue/
+ *
+ * Use draft submission values to populate the draft name.
+ */
+// Update "123" to your form ID.
+add_filter( 'gpasc_draft_display_name_123', function( $display_name, $form_id, $resume_token_data ) {
+	// Update "4" to the ID of the field to be used to set the draft name.
+	$field_id          = 4;
+	$submission_data   = GFFormsModel::get_draft_submission_values( $resume_token_data['token'] );
+	$submission_values = json_decode( $submission_data['submission'], true );
+	$submission_value  = rgar( $submission_values['submitted_values'], $field_id );
+	if ( ! is_array( $submission_value ) ) {
```

Comment:
Can you add a comment here to let users know that this condition would be triggered if using on a single line text field? And then similarly, add a comment to the `else` block with an explanation of how you are grabbing values for the first/last name fields?

These are useful because it sometimes helps others feel confident modifying the scripts themselves for their exact use case.

Reply:
Also, this is not needed to send to the customer to test!

Reply:
Well Noted. I will send it over to the customer to test.